addons/sale_stock/models/account_move.py

```diff
@@ -117,9 +117,8 @@ def _stock_account_get_anglo_saxon_price_unit(self):
         if so_line:
             qty_to_invoice = self.product_uom_id._compute_quantity(self.quantity, self.product_id.uom_id)
             qty_invoiced = sum([x.product_uom_id._compute_quantity(x.quantity, x.product_id.uom_id) for x in so_line.invoice_lines if x.move_id.state == 'posted'])
-            average_price_unit = self.product_id._compute_average_price(qty_invoiced, qty_to_invoice, so_line.move_ids)
+            average_price_unit = self.product_id.with_context(force_company=self.company_id.id)._compute_average_price(qty_invoiced, qty_to_invoice, so_line.move_ids)
```

Comment:
`so_line.company_id.id` ?

Reply:
Same than previous: the object is aml, what does it change to take company from so_line instead of aml?